### DISCUSSION TITLE
How to self-host in Supabase which deprecates pgBouncer?

### DISCUSSION THREAD

**Discussion by GregoryZeng** [umami-software/umami#2462] [Q&A]
The umami self-host guide about Supabase (https://umami.is/docs/running-on-supabase) still uses the pgbouncer connection pool which is yet deprecated by Supabase soon. 
https://supabase.com/docs/guides/database/connecting-to-postgres#supavisor-vs-pgbouncer

I tried to put the supabase's new connection pool URL (`postgres://postgres.xxxxxx:[YOUR-PASSWORD]@aws-0-us-west-1.pooler.supabase.com:6543/postgres` in the `DATABASE_URL` but the `yarn build` fails with error:
```
> umami@2.9.0 check-db
> node scripts/check-db.js

✓ DATABASE_URL is defined.
✓ Database connection successful.
✗
Invalid `prisma.$queryRaw()` invocation:


Raw query failed. Code: `26000`. Message: `ERROR: prepared statement "s0" does not exist`
```

I wonder how to use the new connection pool URL with umami? Thanks!

**Comment by jstansbe** [comment]
Also curious about this.

PgBouncer and IPv4 will be deprecated after 26th January 2024

**Comment by naodeng** [comment]
I also encountered this problem and tried some solutions but the problem was not solved

**Comment by oneminch** [comment]
Make sure the port is accurate in your db string. It's a tiny detail that changed because of the deprecation in my case. It has most likely been changed to `5432` from `6543`.

**Comment by Qnemes** [comment]
Any update on this one?
